src/components/app/data/createAppRouter.jsx

```diff
@@ -27,7 +27,7 @@ export default function createAppRouter(queryClient) {
     createRoutesFromElements(
       <Route
         path="/"
-        element={<AuthenticatedPageRoute><Root /></AuthenticatedPageRoute>}
+        element={<PageWrap><Root /></PageWrap>}
```

Comment:
We _do_ want to support the `?logout=true` case for unauthenticated user. For customers with identity providers, the logout flow will redirect back to Learner Portal route with `?logout=true` where a message with hyperlink to login again is displayed (parity with current state).